lib/YuiRestClient/App.pm

```diff
@@ -84,6 +85,14 @@ sub label {
     });
 }
 
+sub debug_label {
```

Comment:
Just as a note, we don't have such widget, but we might want to have separate class for `YWizard`. From current implementation `YuiRestClient::Widget::Label` has identical implementation.